packages/quantic/force-app/main/default/lwc/quanticNumericFacet/__tests__/quanticNumericFacet.test.js

```diff
@@ -14,63 +13,128 @@ jest.mock('c/quanticUtils', () => ({
   },
   I18nUtils: {
     format: jest.fn(),
+    getLabelNameWithCount: jest.fn(),
   },
 }));
 jest.mock('c/quanticHeadlessLoader');
 
-const selectors = {
-  facetContent: '[data-test="facet-content"]',
-  componentError: 'c-quantic-component-error',
-};
-
 const exampleFacetId = 'example facet id';
 const exampleField = 'exampleField';
+const exampleEngineId = 'exampleEngineId';
 const exampleFilterId = `${exampleFacetId}_input`;
 const defaultOptions = {
+  engineId: exampleEngineId,
   field: exampleField,
-};
-const numericFacetControllerMock = {
-  subscribe: jest.fn((callback) => callback()),
-  state: {
-    facetId: exampleFacetId,
-    values: [],
-  },
+  facetId: exampleFacetId,
+  sortCriteria: 'ascending',
+  numberOfValues: 10,
+  rangeAlgorithm: 'even',
+  label: 'example label',
+  withInput: 'integer',
 };
 const parentFacetIdError = `The ${exampleField} c-quantic-numeric-facet requires dependsOn.parentFacetId to be a valid string.`;
 const expectedValueError = `The ${exampleField} c-quantic-numeric-facet requires dependsOn.expectedValue to be a valid string.`;
 
-function createTestComponent(options = defaultOptions) {
-  prepareHeadlessState();
+const exampleFacetValues = [
+  {start: 1, end: 2, numberOfResults: 10},
+  {start: 2, end: 3, numberOfResults: 10},
+];
 
-  const element = createElement('c-quantic-numeric-facet', {
-    is: QuanticNumericFacet,
-  });
-  for (const [key, value] of Object.entries(options)) {
-    element[key] = value;
-  }
+const initialFacetState = {
+  facetId: exampleFacetId,
+  values: [],
+  enabled: true,
+};
+let facetState = initialFacetState;
 
-  document.body.appendChild(element);
-  return element;
-}
+const initialFilterState = {
+  facetId: exampleFilterId,
+};
+let filterState = initialFilterState;
+
+const initialSearchStatusState = {
+  hasError: false,
+};
+let searchStatusState = initialSearchStatusState;
+
+const selectors = {
+  facetContent: '[data-test="facet-content"]',
```

Comment:
? We should change the html too? Was this a typo?
```suggestion
  facetContent: '[data-testid="facet-content"]',
```